### ISSUE TITLE
error at the end of backup process

### ISSUE THREAD

**Issue by mbravorus** [n1313/ff-backup#1]
After successfully downloading the posts and attachments, process bails out when generating yearly summaries (I believe)

```
Generating 2020 page...
Generating 2021 page...
Generating NaN page...
RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at renderPost (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:200:42)
    at /mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:281:40
    at Array.map (<anonymous>)
    at /mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:281:26
    at Array.forEach (<anonymous>)
    at renderPostPages (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:274:22)
    at buildApp (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:496:3)
    at main (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:503:3)
    at Object.<anonymous> (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:506:1)

**Comment by n1313** [comment]
@mbravorus it looks like one of the posts you have for some reason has an invalid date of creation. I am going to make a tiny little fix to prevent it from blowing up the build process, however if this is not the date that is wrong but the entire post object is corrupted, then it won't really help much. My recommendation is to update the sources after I publish the fix, and re-do the entire backup. If it doesn't help, let me know, we'll work together on identifying the problem.

**Comment by n1313** [comment]
Should be fixed in v1.1.0

**Comment by mbravorus** [comment]
rerun from scratch produced similar result (slightly different, 1970 instead of NaN):

```
> ff-backup@1.1.0 build
> node ./src/build.js

Reading stored backup data for mbravo...
Generating main page....
Generating 1970 page...
RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at renderPost (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:214:42)
    at /mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:295:40
    at Array.map (<anonymous>)
    at /mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:295:26
    at Array.forEach (<anonymous>)
    at renderPostPages (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:288:22)
    at buildApp (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:531:3)
    at main (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:538:3)
    at Object.<anonymous> (/mnt/d/SynologyDrive/Sources/ff-backup/src/build.js:541:1)

**Comment by n1313** [comment]
Should be fixed (for real this time) in v1.1.1

**Comment by mbravorus** [comment]
Confirmed, after updating to v1.1.1, build phase completed successfully. Thanks!
